spec/exception_handler_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "exception_handler"
+
+RSpec.describe ExceptionHandler do
+  describe ".capture" do
+    it "sends any exceptions to Sentry and doesn't let them propagate" do
+      stub_const("SilencedError", Class.new(StandardError))
+
+      allow(Sentry).to receive(:capture_exception)
+
+      expect {
+        described_class.capture do
+          raise SilencedError, "This error is sent to Sentry and silenced"
+        end
+      }.not_to raise_error
+
+      expect(Sentry).to have_received(:capture_exception)
+        .with(an_instance_of(SilencedError))
+    end
+  end
+
+  describe ".report" do
```

Comment:
This might be better named propogate? I keep checking the difference between `report` and `capture`

Reply:
I agree with you that `report` and `capture` are bad names. And the more I look at them, probably also that `ExceptionHandler.propagate` and `ExceptionHandler.capture` are a better pair. But I hope we can do even better :laughing: